py/desisim/scripts/quickquasars.py

```diff
@@ -61,7 +66,8 @@ def parse(options=None):
 
     parser.add_argument('--sigma_kms_fog',type=float,default=150, help="Adds a gaussian error to the quasar redshift that simulate the fingers of god effect")
 
-    parser.add_argument('--sigma_kms_zfit',nargs='?',type=float,const=400,help="Adds a gaussian error to the quasar redshift, to simulate the redshift fitting step. E.g. --sigma_kms_zfit 200 will use a sigma value of 200 km/s. If a number is not specified the default is used.")
+    parser.add_argument('--sigma_kms_zfit',nargs='?',type=float,const=200,help="Adds a Lorentzian distributed shift to the quasar redshift, to simulate the redshift fitting step. E.g. --sigma_kms_zfit 200 will use a sigma value of 200 km/s. If a number is not specified the default is used.")
```

Comment:
Hi @alxogm - Last minor request: should we change the name sigma_kms_zfit to something else? If I read sigma_kms_fog and then sigma_kms_zfit, I have the impression that both are talking about Gaussian RMS, but the latter is not a Gaussian rms. Why call it "sigma"? What is the usual term used to refer to the width of a Lorentzian? gamma?